fmn/api/distgit.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from fastapi import Depends
+from httpx import AsyncClient
+
+from ..core.config import Settings, get_settings
+
+log = logging.getLogger(__name__)
+
+
+class DistGitClient:
+    def __init__(self, settings):
+        self.client = AsyncClient(base_url=settings.services.distgit_url, timeout=None)
+
+    async def get_artifacts(self, pattern):
+        response = await self.client.get(
+            "/api/0/projects",
+            params={
+                "pattern": f"*{pattern}*",
+                "short": "true",
+            },
+        )
+        response.raise_for_status()
+        result = response.json()
```

Comment:
In the results I get all the forks of the search projects. Is there a way to query distgit to exclude the forks? (I tried searching for `0`).

Reply:
whoops! i forgot to add this in (my test data doesnt have any forks, so missed this)

Reply:
fixed!